x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/file_data_visualizer_view/file_error_callouts.tsx

```diff
@@ -146,3 +146,26 @@ export const Explanation: FC<{ error: FindFileStructureErrorResponse }> = ({ err
     </>
   );
 };
+
+export const FindFileStructurePermissionDenied: FC = () => {
+  return (
+    <>
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.dataVisualizer.file.fileErrorCallouts.findFileStructurePermissionDenied.title"
+            defaultMessage="Permission denied"
+          />
+        }
+        color="danger"
+        iconType="cross"
+        data-test-subj="dataVisualizerFileStructurePermissionDeniedErrorCallout"
+      >
+        <FormattedMessage
+          id="xpack.dataVisualizer.file.fileErrorCallouts.findFileStructurePermissionDenied.description"
+          defaultMessage="monitor_text_structure or monitor cluster privileges are required to use this feature"
```

Comment:
In https://elastic.github.io/eui/#/guidelines/writing, the general advice is to address users as "you", which I agree feels friendlier:
```suggestion
          defaultMessage="You must have monitor_text_structure or monitor cluster privileges to use this feature."
```